Backoffice/GenerateForm/Strategies/VideoStrategy.php

```diff
@@ -12,7 +12,7 @@
  */
 class VideoStrategy extends AbstractBlockStrategy
 {
-    const TEMPLATE = 'OpenOrchestraBackofficeBundle:Block/Video:form.html.twig';
+    const TEMPLATE = 'OpenOrchestraBackofficeBundle:Block:Video/Form/form.html.twig';
```

Comment:
Both are used on the project but I think it's a good idea to have the same logic everywhere. I'm ok with your proposal.